packages/manager/src/components/CircleProgress/index.ts

```diff
@@ -1,2 +1,2 @@
-import CircleProgress from './CircleProgress';
-export default CircleProgress;
+import { CircleProgress } from './CircleProgress';
+export { CircleProgress };
```

Comment:
We should be able to write this as one line instead of two:
```suggestion
export { CircleProgress } from './CircleProgress';
```